big_scape/cli/cluster_cli.py

```diff
@@ -35,7 +35,17 @@
     ),
 )
 # binning parameters
-@click.option("--no_mix", is_flag=True, help=("Don't run the all-vs-all analysis."))
+@click.option(
+    "--mix",
+    is_flag=True,
+    help=(
+        "Calculate distances using a 'mix' bin, wherein no classification is applied. "
+        "This will do an all-vs-all comparison, and is likely going to take a long time. "
+        "This bin will use weights from the 'mix' weights distribution. Warning: these "
+        "weights are not recommended for use with the record types protocluster/protocore, "
```

Comment:
no need to include the weights disclaimer here. mix weights are for mix bin, whichever record type is being used. same behaviour as v1.